packages/elements/src/item/index.ts

```diff
@@ -204,6 +219,44 @@ export class Item extends ControlElement {
     }
   }
 
+
+  /**
+   * Get Item content
+   * @returns return item content from slot or label and sub-label
+   */
+  private getItemContent (): string {
+    if (this.isSlotEmpty) {
```

Comment:
Why not to play around `render` function to simplify the logic? For instance:

```
<div part="center" id="label">
        <div part="label" ref="....">
          ${ isSlotEmpty ? this.labelTemplate : undefined }
          <slot @slotchange="${this.checkSlotChildren}"></slot>
        </div>
        ${ isSlotEmpty ? this.subLabelTemplate : undefined }
      </div>
```

and the you just get textContent form labelRef and subLabelRef.

Reply:
@Nantawat-Poothong can you comment on this?

Reply:
@TremayneChrist Sorry for the late comment, yes I think it's possible.

Reply:
Maybe we could simplify logic like this?
```javascript
<div part="center" id="label">
  <div part="label" ref="....">
    ${ isSlotEmpty && this.label ? this.labelTemplate : undefined }
    <slot @slotchange="${this.checkSlotChildren}"></slot>
  </div>
  ${ isSlotEmpty && this.subLabel ? this.subLabelTemplate : undefined }
</div>
````

Reply:
Yeah let's try

Reply:
Updated.